client/data/authorizations/resolvers.ts

```diff
@@ -0,0 +1,93 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { dateI18n } from '@wordpress/date';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import { updateAuthorizations, updateAuthorizationsSummary } from './actions';
+import { Authorization } from './hooks';
+import { Query } from '@woocommerce/navigation';
+
+enum RiskLevel {
```

Comment:
I've seen some recommendations to avoid using `enum` in TS. You can find more info here: p7H4VZ-3sK-p2. It recommends to use a union of string literals instead, i.e:

`type RiskLevel = 'high' | 'elevated' | 'normal'`

Reply:
Aha, good to know! Thanks! I fixed those